src/GenAPI/Microsoft.DotNet.GenAPI/Filters/IntersectionFilter.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions;
+
+namespace Microsoft.DotNet.GenAPI
+{
+    internal class IntersectionFilter : ISymbolFilter
```

Comment:
This essentially is the composite pattern: https://en.wikipedia.org/wiki/Composite_pattern.

The English word "intersection" means something different than a composite. Hence, I would call it `CompositeFilter`.

Reply:
Can you please also share this filter with ApiCopmat via ApiSymbolExtensions? As part of that, when you make the type public, please also add triple slash xml documentation to the type itself.